elevation_mapping/include/elevation_mapping/ElevationMapping.hpp

```diff
@@ -256,6 +256,9 @@ class ElevationMapping
 
   //! Callback thread for raytracing cleanup.
   boost::thread visibilityCleanupThread_;
+
+  //! Becomes true when corresponding poses and point clouds can be found
+  bool initialized_; 
```

Comment:
I think the name initialized is a bit misleading, as it should be false before calling initialize() and true afterwards. Rename?